tests/src/tracing/eventpipe/rundownvalidation/rundownvalidation.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Microsoft.Diagnostics.Tools.RuntimeClient;
+using Microsoft.Diagnostics.Tracing;
+using Tracing.Tests.Common;
+
+namespace Tracing.Tests.RundownValidation
+{
+
+    public class RundownValidation
+    {
+        private static Dictionary<string, int> _expectedEventCounts = new Dictionary<string, int>()
+        {
+            { "Microsoft-Windows-DotNETRuntimeRundown", -1 }
+        };
+
+        private static Action _eventGeneratingAction = () => 
+        {
+            Thread.Sleep(500);
```

Comment:
If the goal is getting rundown events, why does the test need to idle? Returning immediately still has rundown events.